common/lib/capa/capa/templates/formulaequationinput.html

```diff
@@ -4,7 +4,7 @@
 <div id="formulaequationinput_${id}" class="inputtype formulaequationinput" ${doinline | n, decode.utf8}>
     <div class="${status.classname}" id="status_${id}">
       % if response_data['label']:
-          <label class="problem-group-label" for="input_${id}">${response_data['label']}</label>
+          <label class="problem-group-label" for="input_${id}">${HTML(response_data['label'])}</label>
       % endif
         % for description_id, description_text in response_data['descriptions'].items():
             <p class="question-description" id="${description_id}">${description_text}</p>
```

Comment:
1. Whatever you do here isn't really tested until you add default HTML escaping using `<%page expression_filter="h"/>`.
2. You may consider moving the `HTML()` call to where response_data['label'] is being set, in place of having it here.  I'm not sure which will be cleaner.
3. Ideally, `response_data['label']` would be renamed to `response_data['label_html']`, which would make all of this more clear.